packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -101,18 +123,46 @@ export class AtomicFacet implements InitializableComponent {
     };
     this.facet = buildFacet(this.bindings.engine, {options});
     this.facetId = this.facet.state.facetId;
-
-    this.strings[this.label] = () => this.bindings.i18n.t(this.label);
     this.bindings.store.state.facets[this.facetId] = {
       label: this.label,
     };
   }
 
-  private get shouldDisplaySearch() {
-    return this.withSearch && this.facetState.canShowMoreValues;
+  private get numberOfSelectedValues() {
+    return this.facetState.values.filter(({state}) => state === 'selected')
+      .length;
+  }
+
+  private renderHeader() {
+    return (
+      <FacetHeader
```

Comment:
Yes, a bit of repetition, mainly because of i18n we need a state property. I'll investigate passing the strings state as prop see if we can add them manually inside of the functional components 🤔